proactive-jupyter-kernel/kernel.py

```diff
@@ -798,13 +800,19 @@ def __submit_job__(self, input_data):
 
         self.__kernel_print_ok_message__('Submitting the job to the proactive scheduler ...\n')
 
-        self.submitted_jobs_names.append(self.job_name)
-        self.submitted_jobs_ids[self.job_name] = self.gateway.submitJob(self.proactive_job, debug=False)
+        temp_id = self.gateway.submitJob(self.proactive_job, debug=False)
+        self.submitted_jobs_names[temp_id] = self.job_name
+        self.submitted_jobs_ids[self.job_name] = temp_id
 
-        self.__kernel_print_ok_message__('job_id: ' + str(self.submitted_jobs_ids[self.job_name]) + '\n')
+        self.__kernel_print_ok_message__('job_id: ' + str(temp_id) + '\n')
```

Comment:
Sometimes you use underscores, camel cases, and/or double underscores. I wonder if you follow any standard for the nomenclature used in this project. Thanks for clarifying :)

Reply:
Methods declared with underscore are methods that are meant to be private. It is a kind of a formalism in python since there is not the private feature. I used this way of declaring the main methods of the Kernel.py file/class.